extemporaneous_grading/static/html/children.html

```diff
@@ -0,0 +1,28 @@
+{% load i18n %}
+<div class="extemporaneous_grading_block">
+    <div class="dates">
+        <span><b>{% trans "Due Date: " %}</b>{{ block.due_datetime }} UTC</span>
+        <span><b>{% trans "Late Due Date: " %}</b>{{ block.late_due_datetime }} UTC</span>
+    </div>
+    <hr />
+    {% for child_content in children_contents %}
+        {{ child_content|safe }}
+        <br />
+        <hr />
+    {% endfor %}
+</div>
+
+<script type="text/javascript" src="/extemporaneous_grading/static/js/src/resize_iframe.js"></script>
```

Comment:
I thought this was being added here regardless: https://github.com/eduNEXT/xblock-extemporaneous-grading/pull/2/files#diff-e60628835ced9b5ca0bccf5fd0ed4e046442e626c2fc5cd9e8127555307fc1d7R244
Should this be done manually too?

Reply:
That's right! We don't need to add it manually. Thank you!

Done: https://github.com/eduNEXT/xblock-extemporaneous-grading/commit/0b0a990f9fb3e1630c7dd8e1673fe1eca47f1002